solr/solrj/src/test/org/apache/solr/client/solrj/SolrExampleTests.java

```diff
@@ -602,58 +602,60 @@ public void testUnicode() throws Exception {
     SolrClient client = getSolrClient();
 
     // save the old parser, so we can set it back.
-    ResponseParser oldParser = null;
-    if (client instanceof HttpSolrClient) {
-      HttpSolrClient httpSolrClient = (HttpSolrClient) client;
-      oldParser = httpSolrClient.getParser();
-    }
-
-    try {
-      for (int iteration = 0; iteration < numIterations; iteration++) {
-        // choose format
-        if (client instanceof HttpSolrClient) {
-          if (random.nextBoolean()) {
-            ((HttpSolrClient) client).setParser(new BinaryResponseParser());
-          } else {
-            ((HttpSolrClient) client).setParser(new XMLResponseParser());
-          }
-        }
-
-        int numDocs = TestUtil.nextInt(random(), 1, 10 * RANDOM_MULTIPLIER);
-
-        // Empty the database...
-        client.deleteByQuery("*:*"); // delete everything!
-
-        List<SolrInputDocument> docs = new ArrayList<>();
-        for (int i = 0; i < numDocs; i++) {
-          // Now add something...
-          SolrInputDocument doc = new SolrInputDocument();
-          doc.addField("id", "" + i);
-          doc.addField("unicode_s", randomTestString(30));
-          docs.add(doc);
-        }
+    // ResponseParser oldParser = null;
+    // if (client instanceof HttpSolrClient) {
+    //  HttpSolrClient httpSolrClient = (HttpSolrClient) client;
+    //  oldParser = httpSolrClient.getParser();
+    // }
```

Comment:
Guess still figuring out how this tests works/needs to be converted?

Reply:
Actually hoping someone else can confirm that the changes I made still keep the test working "as designed", and that this commented out code can just be deleted....

Reply:
Not sure where my other comment went :) I had one typed up about this but looks like it went poof.

Since the test is called `testUnicode`  - I wonder if `SolrExampleBinaryTest` or `SolrExampleXMLTest` test unicode? maybe this is specific to unicode and thats why the parser is randomized?